### ISSUE TITLE
@elseif is not recognized

### ISSUE THREAD

**Issue by mzgol** [sass/libsass#1060]
The following SCSS:
```scss
div {
    @if true {
        display: block;
    } @elseif true {
        display: flex;
    }
}
```
is translated via libsass `3.2.0-beta.5` to:
```css
div {
  display: block; }

@elseif true {
  div {
    display: flex; } }
```
Ruby Sass translates it to:
```css
div {
  display: block; }
```
Changing `@elseif` to `@else if` resolves the issue.

**Comment by xzyfer** [comment]
Specs added https://github.com/sass/sass-spec/pull/306

**Comment by xzyfer** [comment]
This is fixed and will be 3.2.0

**Comment by mzgol** [comment]
Thanks! :)
